From c2b5650f238ca9d2651070db69cebb811043db54 Mon Sep 17 00:00:00 2001
From: 1xtAsh <ashraykrsingh321@gmail.com>
Date: Wed, 2 Oct 2024 23:03:45 +0530
Subject: [PATCH] sm8350-common: Blaze-ify!

---
 BoardConfigCommon.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 0883822..21ae549 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -68,7 +68,7 @@ DEVICE_MATRIX_FILE := $(COMMON_PATH)/hidl/compatibility_matrix.xml
 DEVICE_FRAMEWORK_COMPATIBILITY_MATRIX_FILE := \
     $(COMMON_PATH)/hidl/vendor_framework_compatibility_matrix.xml \
     $(COMMON_PATH)/hidl/xiaomi_framework_compatibility_matrix.xml \
-    vendor/lineage/config/device_framework_matrix.xml
+    vendor/blaze/config/device_framework_matrix.xml
 
 DEVICE_MANIFEST_FILE := \
     $(COMMON_PATH)/hidl/manifest_lahaina.xml \
@@ -125,7 +125,7 @@ BOARD_USERDATAIMAGE_FILE_SYSTEM_TYPE := f2fs
 BOARD_VENDORIMAGE_FILE_SYSTEM_TYPE := ext4
 BOARD_VENDOR_DLKMIMAGE_FILE_SYSTEM_TYPE := ext4
 
--include vendor/lineage/config/BoardConfigReservedSize.mk
+-include vendor/blaze/config/BoardConfigReservedSize.mk
 BOARD_SYSTEM_EXTIMAGE_PARTITION_RESERVED_SIZE := 104857600
 BOARD_VENDORIMAGE_PARTITION_RESERVED_SIZE := 104857600
 
-- 
2.34.1

