From ac07a596b9a22d9441c486ddcb40d0311feb6948 Mon Sep 17 00:00:00 2001
From: 1xtAsh <ashraykrsingh321@gmail.com>
Date: Thu, 31 Oct 2024 13:26:48 +0530
Subject: [PATCH] sm8350-common: Derpify!

---
 BoardConfigCommon.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 83f34af..8935d04 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -68,7 +68,7 @@ DEVICE_MATRIX_FILE := $(COMMON_PATH)/hidl/compatibility_matrix.xml
 DEVICE_FRAMEWORK_COMPATIBILITY_MATRIX_FILE := \
     $(COMMON_PATH)/hidl/vendor_framework_compatibility_matrix.xml \
     $(COMMON_PATH)/hidl/xiaomi_framework_compatibility_matrix.xml \
-    vendor/lineage/config/device_framework_matrix.xml
+    vendor/derp/config/device_framework_matrix.xml
 
 DEVICE_MANIFEST_FILE := \
     $(COMMON_PATH)/hidl/manifest_lahaina.xml \
@@ -125,7 +125,7 @@ BOARD_USERDATAIMAGE_FILE_SYSTEM_TYPE := f2fs
 BOARD_VENDORIMAGE_FILE_SYSTEM_TYPE := ext4
 BOARD_VENDOR_DLKMIMAGE_FILE_SYSTEM_TYPE := ext4
 
--include vendor/lineage/config/BoardConfigReservedSize.mk
+-include vendor/derp/config/BoardConfigReservedSize.mk
 BOARD_SYSTEM_EXTIMAGE_PARTITION_RESERVED_SIZE := 104857600
 BOARD_VENDORIMAGE_PARTITION_RESERVED_SIZE := 104857600
 
-- 
2.34.1

